client/marklogicClient.ts

```
@@ -340,13 +349,16 @@ export async function getFilteredListOfJsAppServers(
     return await filterJsServerByConnectedStatus(dbClientContext, managePort, allPossibleJsServers, requiredResponse);
 }
 
-export async function getAppServerListForJs(dbClientContext: ClientContext, serverListQuery: string): Promise<MlxprsQuickPickItem[]> {
+export async function getAppServerListForJs(
+    dbClientContext: ClientContext, serverListQuery: string
+): Promise<MlxprsQuickPickItem[]> {
     return sendXQuery(dbClientContext, serverListQuery)
         .result(
             (appServers: Record<string, ServerQueryResponse>) => {
                 return dbClientContext.buildAppServerChoicesFromServerList(appServers);
             },
             err => {
+                console.error('Error: ' + JSON.stringify(err));
```

Is this adding at least something so that the error isn't silently ignored?